interface-definitions/include/firewall/add-dynamic-address-groups.xml.i

```diff
@@ -0,0 +1,49 @@
+<!-- include start from firewall/add-dynamic-address-groups.xml.i -->
+<leafNode name="address-group">
+  <properties>
+    <help>Dynamic address-group</help>
+    <completionHelp>
+      <path>firewall group dynamic-group address-group</path>
+    </completionHelp>
+  </properties>
+</leafNode>
+<leafNode name="timeout-value">
+  <properties>
+    <help>Set timeout</help>
+    <valueHelp>
+      <format>u32:1-255</format>
+      <description>Remains element in group for this time</description>
+    </valueHelp>
+    <constraint>
+      <validator name="numeric" argument="--range 1-255"/>
+    </constraint>
+  </properties>
+</leafNode>
+<leafNode name="timeout-unit">
```

Comment:
I always find CLIs with `foo-x` and `foo-y` nodes at the same level aesthetically unappealing, to be fair. In any case I suppose we need to agree on how we handle commands for time intervals that can be measured in days (and thus are impractical to specify in seconds).
In `system task-scheduler`, it's `interval \d+(s|m|d)`.
I'd like to hear other opinions on that.